PicPi/photosUpload.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  photosUpload.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 12/16/20.
+//
+
+import Foundation
+import Moya
+
+/// photosUpload endpoint
+enum photosUpload {
+  ///enter nameAndExtension as one entry because when using PHAsset will return full file name with extension 
+  case uploadPhoto(photoData: Data , nameAndExtension: String)
+}
+
+extension photosUpload: TargetType {
+  var baseURL: URL {
+    ///Pi IP hard coded for now
+     return URL(string: "http://192.168.0.5:3000")!
+  }
+  /// path for the specified operation (e.g. : upload photo , delete photo...)
+  var path: String {
+    switch self {
+      case .uploadPhoto(_, _):
```

Comment:
nit: if none of the values of the enum are used you can not include it like so:

```suggestion
      case .uploadPhoto:
```